lib/intercom_flutter.dart

```diff
@@ -162,4 +162,13 @@ class Intercom {
     return await _channel
         .invokeMethod<void>('handlePush', {'message': message});
   }
+
+  /// override the language of Intercom messenger.
+  ///
+  /// The [language] should be an an ISO 639-1 two-letter code such as 'en' for English or 'fr' for French.
+  /// You’ll need to use a four-letter code for Chinese like 'zh-CN'.
```

Comment:
I think this might not be only for Chinese, there are more specific locales that work like this, but I'm not sure how many of those Intercom supports. A link to supported languages can be very useful here

Reply:
I have added the Intercom link.